### PR TITLE
Docs: Added bookmarklet method to import CKEditor 5 inspector

### LINKED ISSUE #12032: Add inspector bookmarklet code to the development tools/inspector section
## 📝 Provide a description of requested docs changes

ATM [the section](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/development-tools.html#ckeditor-5-inspector) only contains information inspector imported from npm or using `<script>`. There's another option: a bookmarklet that we use every day in the team that allows running inspector on any webpage without modifying its source.

```js
javascript:(function(){let script=document.createElement('script');script.src='https://unpkg.com/@ckeditor/ckeditor5-inspector/build/inspector.js';script.onload=()=>CKEditorInspector.attachToAll();document.head.appendChild(script);})()
```

### PR DESCRIPTION
**Pull request by dufipl**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs: Added bookmarklet method to import CKEditor 5 inspector. Closes #12032.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/framework/guides/development-tools.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ or as a plain `<script>` tag in the HTML of your application:
 <script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
 ```
 
+#### Inspector as a bookmarklet
+
+Alternatively, you might create a bookmarklet in your browser that will allow you to open inspector on every page without interference
```
Comment (godai78) [inline docs/framework/guides/development-tools.md:40]:
```suggestion
Alternatively, you can create a bookmarklet in your browser that will allow you to open inspector on any page without interference with its source code. 
```

File: docs/framework/guides/development-tools.md [hunk 2]
```diff
@@ -35,6 +35,17 @@ or as a plain `<script>` tag in the HTML of your application:
...
+ with its source code.
```
Comment (godai78) [inline docs/framework/guides/development-tools.md:41]: this line should be removed

File: docs/framework/guides/development-tools.md [hunk 3]
```diff
@@ -35,6 +35,17 @@ or as a plain `<script>` tag in the HTML of your application:
 <script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
 ```
 
+#### Inspector as a bookmarklet
+
+Alternatively, you might create a bookmarklet in your browser that will allow you to open inspector on every page without interference
+ with its source code. 
+<b>Important note: this method won't work if the page has Content Security Policy enabled.</b>
```
Comment (godai78) [inline docs/framework/guides/development-tools.md:42]:
```suggestion
<b>Important note: this method will not work if the page has Content Security Policy enabled.</b>
```

File: docs/framework/guides/development-tools.md (hunk 1 above)
Comment (Reinmar) [inline docs/framework/guides/development-tools.md:40]: Some "markdown-codestyle" things:

* Don't wrap the lines manually. Configure your IDE to wrap them for you.
* Replace `<b>` with `**`.
* Add extra padding (empty new lines) between blocks of content. In this case – before and after the code block.

File: docs/framework/guides/development-tools.md [hunk 4]
```diff
@@ -35,6 +35,17 @@ or as a plain `<script>` tag in the HTML of your application:
...
+To create a bookmarklet, paste this code as the URL of the newly created bookmark in the browser of your choice:
+```js
+javascript:(function(){let script=document.createElement('script');script.src='https://unpkg.com/@ckeditor/ckeditor5-inspector/build/inspector.js';script.onload=()=>CKEditorInspector.attachToAll();document.head.appendChild(script);})()
+```
+After that, you can open CKEditor 5 inspector by pressing the previously created bookmark.
```
Comment (Reinmar) [inline docs/framework/guides/development-tools.md:47]: open -> load?

Opening sounds like it's closed so you open it. While with inspector it's more to make it appear at all.

File: docs/framework/guides/development-tools.md (hunk 1 above)
Comment (godai78) [reply docs/framework/guides/development-tools.md]: Done.

File: docs/framework/guides/development-tools.md (hunk 4 above)
Comment (godai78) [reply docs/framework/guides/development-tools.md]: Also changed pressing -> using.

File: docs/framework/guides/development-tools.md [hunk 5]
```diff
@@ -35,6 +35,20 @@ or as a plain `<script>` tag in the HTML of your application:
 <script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
 ```
 
+#### Inspector as a bookmarklet
```
Comment (Reinmar) [inline docs/framework/guides/development-tools.md:38]: I know it sounds a bit like a sub-section to "Importing the inspector" but heading level 4 is really deeeep in the structure. So maybe l3 looks better?

**Review by Mgsy — APPROVED** [review]

**Pull request merged**